### PR TITLE
[RHACS] ROX-10950: Updating content type variable in ACS documentation (rhacs-docs and 4.3)

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.3+

Issue:
https://issues.redhat.com/browse/ROX-10950

Link to docs preview:

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs` and cherry-pick to:
    - `rhacs-docs-4.3`

### CODE REVIEW

File: modules/automatic-upgrade-failure.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 //
 // * configuration/configure-automatic-upgrades.adoc
-:_module-type: CONCEPT
+:_mod-docs-content-type:CONCEPT
```
Comment (kalexand-rh) [inline modules/automatic-upgrade-failure.adoc:4]: You need a space here.
```suggestion
:_mod-docs-content-type: CONCEPT
```

It looks like all of the ones that were ":_content-type:" are correct, but the ones that were ":_module-type:"  all need a space.

**Review by gaurav-nelson — APPROVED** [review]

**Comment by gaurav-nelson** [comment]
**suggestion:** @agantony I've approved the PR, however, please rebase and combine the commits into one before merging. Thank you.

**Pull request merged**